src/app/templates/registration/password_reset_email.html

```diff
@@ -0,0 +1,12 @@
+{% autoescape off %}
+To initiate the password reset process for your {{ user.get_username }} SPDX Online tools Account,
+click the link below:
+
+{{ protocol }}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}
+
+If clicking the link above doesn't work, please copy and paste the URL in a new browser
+window instead.
+
```

Comment:
If you have received this email and have not requested to reset the password for this user account, please contact < test@spdx.org > immediately regarding this unexpected activity.